config/v1/0000_10_config-operator_01_scheduler.crd.yaml

```diff
@@ -19,89 +19,90 @@ spec:
   - name: v1
     served: true
     storage: true
+    "schema":
```

Comment:
remove preserveUnknownFields above

Reply:
/lgtm cancel
btw, we check this isn't set in verify-crds.sh but only for v1beta1 CRDs. should we expand that to check that it's not set in v1? I thought that wasn't even available in v1

Reply:
Yes, please.

Also top-level subresources is a common error. I also did it in the past.